x-pack/plugins/data_visualizer/public/application/common/components/stats_table/_index.scss

```diff
@@ -2,55 +2,90 @@
 @import 'components/field_count_stats/index';
 @import 'components/field_data_row/index';
 
-.dataVisualizerFieldExpandedRow {
+$panelWidthS:     #{'max(20%, 225px)'};
+$panelWidthL:     #{'max(40%, 450px)'};
+
+.dvExpandedRow {
   padding-left: $euiSize * 4;
   width: 100%;
 
-  .fieldDataCard__valuesTitle {
+  .dvExpandedRow__fieldHeader {
     text-transform: uppercase;
     text-align: left;
     color: $euiColorDarkShade;
     font-weight: bold;
     padding-bottom: $euiSizeS;
   }
-
-  .fieldDataCard__codeContent {
-    @include euiCodeFont;
-  }
 }
 
-.dataVisualizer {
-  .euiTableRow > .euiTableRowCell {
-    border-bottom: 0;
-    border-top: $euiBorderThin;
+@include euiBreakpoint('m', 'l', 'xl') {
+  .dvTable {
+    .columnHeader__title {
+      display: flex;
+      align-items: center;
+    }
 
-  }
-  .euiTableRow-isExpandedRow {
+    .columnHeader__icon {
+      padding-right: $euiSizeXS;
+    }
+
+    .euiTableRow > .euiTableRowCell {
+      border-bottom: 0;
+      border-top: $euiBorderThin;
 
-    .euiTableRowCell {
-      background-color: $euiColorEmptyShade !important;
-      border-top: 0;
-      border-bottom: $euiBorderThin;
-      &:hover {
+    }
+
+    .euiTableCellContent {
+      padding: 4px;
```

Comment:
You'll need to replace this with an Eui variable.

Reply:
Updated here https://github.com/elastic/kibana/pull/112870/commits/858553061dde9609eccf4d50e4ef37ab80391c18